app/policies/project_policy.rb

```diff
@@ -0,0 +1,36 @@
+class ProjectPolicy < ApplicationPolicy
+  def editor?
+    return false if records.empty?
+    records.compact.all? do |record|
```

Comment:
As the records are passed into the `authorize` method, why would the records be empty here? If really needed, could we compact at the `initializer` of the `ApplicationPolicy` class?

Reply:
Fair, probably not necessary to check.